src/main/java/com/bunq/sdk/context/ApiContext.java

```diff
@@ -91,19 +95,36 @@ private ApiContext(ApiEnvironmentType environmentType, String apiKey) {
   }
 
   /**
-   * Create and initialize an API Context with current IP as permitted.
+   * Create and initialize an API Context with current IP as permitted and no proxy.
    */
   public static ApiContext create(ApiEnvironmentType environmentType, String apiKey,
       String deviceDescription) {
     return create(environmentType, apiKey, deviceDescription, new ArrayList<>());
   }
 
   /**
-   * Create and initialize an API Context.
+   * Create and initialize an API Context with given permitted ips and no proxy.
    */
   public static ApiContext create(ApiEnvironmentType environmentType, String apiKey,
       String deviceDescription, List<String> permittedIps) {
+    return create(environmentType, apiKey, deviceDescription, permittedIps, null);
+  }
+
+  /**
+   * Create and initialize an API Context with current IP as permitted and a proxy.
+   */
+  public static ApiContext create(ApiEnvironmentType environmentType, String apiKey,
+      String deviceDescription, String proxy) {
+    return create(environmentType, apiKey, deviceDescription, new ArrayList<>(), proxy);
+  }
+
+  /**
+   * Create and initialize an API Context.
```

Comment:
Hmm should this then be something like `Create and initialize an API Context with given permitted ips and given proxy.` ?